collect_app/src/test/java/org/odk/collect/android/formentry/QuitFormDialogFragmentTest.java

```diff
@@ -66,6 +67,21 @@ public void shouldShowCorrectTitle() {
         assertThat(dialogTitle.getText().toString(), equalTo(title));
     }
 
+
+    @Test
+    public void shouldShowCorrectTitle_whenFormIsLoaded() {
+        when(formSaveViewModel.getFormName()).thenReturn("blah");
+
+        dialogFragment.show(fragmentManager, "TAG");
+        AlertDialog dialog = (AlertDialog) ShadowDialog.getLatestDialog();
+
+        String title =  formSaveViewModel.getFormName() == null ? activity.getString(R.string.no_form_loaded)
```

Comment:
You don't need this logic in either test right? The test knows what value the form name will be.

Reply:
Ah sorry! This piece of code is not required.